pyked/schemas/chemked_schema.yaml

```
@@ -28,7 +29,8 @@ common-properties:
     pressure: *value-unit-optional
     ignition-type: *ignition-type
     composition: *composition
-    pressure-rise: *value-unit-optional
+    reactor-volume: *value-unit-optional
```

```suggestion
    pressure-rise: *value-unit-optional
    reactor-volume: *value-unit-optional
```
Don't remove the pressure-rise option